test/test_cuda.py

```diff
@@ -274,6 +274,11 @@ def test_set_per_process_memory_fraction(self):
         tensor.fill_(1)
         self.assertTrue((tensor == 1).all())
 
+    def test_uuid(self):
+        uuid = torch.cuda.get_device_properties(0).uuid
+        assert len(str(uuid)) == 36  # xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+        assert len(uuid.bytes) == 16
```

Comment:
Ho completely missed that... We should use self.assertEqual here. That would have given us much nice error message to work with from the test job!

Reply:
Thanks for the suggestion.  I have changed it to self.asserEqual now.